src/users/models.py

```diff
@@ -33,7 +33,11 @@ class Mentor(models.Model):
     major = models.ForeignKey(Major, null=True, on_delete=models.SET_NULL)
     bio = models.CharField(max_length=5000, null=False, blank=True, default = '')
     active = models.BooleanField(default=True)
-
+    gpa = models.DecimalField(default=0.00, max_digits = 4, decimal_places =2)
+    clubs = models.CharField(max_length=500, null=False, blank=True, default = '')
+    classes = models.CharField(max_length=5000, null=False, blank=True, default = '')
+    pros =  models.CharField(max_length=5000, null=False, blank=True, default = '')
+    cons =  models.CharField(max_length=5000, null=False, blank=True, default = '')
```

Comment:
for keyword args as in 
`function(a, b, c=d)`, the keyword arguments have no space between the arg, =, and value

Reply:
also, are we doing just a paragraph for pros/cons?